Gemfile

```diff
@@ -57,7 +57,7 @@ group :development do
 end
 
 group :test do
-  gem 'capybara', '>= 2.15'
+  gem 'capybara', '~> 3.35'
```

Comment:
We were already on capybara 3.35.3 (which is the latest version, so I updated the version here to make it clear that we're not on 2.x).